client/src/state/mobileMenu/MobileMenuReducer.js

```diff
@@ -0,0 +1,26 @@
+import deepFreeze from 'deep-freeze-strict';
+import ACTION_TYPES from './MobileMenuActionTypes';
+
+const initialState = deepFreeze({});
+
+function reducer(state = initialState, action) {
+  const isOpen = !!(state && state.isOpen);
+
+  switch (action.type) {
+    case ACTION_TYPES.TOGGLE_MENU:
+      return deepFreeze(Object.assign({}, state, { isOpen: !isOpen }));
+
+    case ACTION_TYPES.OPEN_MENU:
+      return deepFreeze(Object.assign({}, state, { isOpen: true }));
+
+    case ACTION_TYPES.CLOSE_MENU:
+      return deepFreeze(Object.assign({}, state, { isOpen: false }));
+
+    default:
+      return state;
+
```

Comment:
We now have object spreads enabled on this module. We can start deprecating `Object.assign` in favour of ` { ...state, isOpen: false }`.